app/src/main/java/com/github/factotum_sdp/factotum/ui/display/DisplayFragment.kt

```diff
@@ -175,19 +175,27 @@ class DisplayFragment : Fragment(), MenuProvider {
         binding.refreshButton.setOnClickListener {
             courierBossDisplayViewModel.refreshFolders()
         }
-
-        val courierBossFolderAdapter = CourierBossFolderAdapter(
-            onCardClick = { clientFolder ->
-                userFolder.value = clientFolder.value
-                observeClientPhotos()
-                setupClientUI()
-            }
-        )
+        val proofPicture = arguments?.getString("ProofPicture")
```

Comment:
Maybe change the "ProofPicture" value to a constant, so it may be easier to switch in the future.